independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/BootstrapConstants.java

```diff
@@ -6,12 +6,12 @@
  */
 public interface BootstrapConstants {
 
-    String SERIALIZED_APP_MODEL = "serialized-app-model.data";
+    String SERIALIZED_APP_MODEL = "quarkus.serialized-app-model.path";
```

Comment:
I wouldn't make it `quarkus.` prefixed as we might have some warning coming. Better use `quarkus-internal.` that we already use a few times in the code.